research/object_detection/core/preprocessor.py

```diff
@@ -1711,15 +1714,16 @@ def _compute_new_dynamic_size(image, min_dimension, max_dimension):
         lambda: small_size, lambda: large_size)
   else:
     new_size = large_size
-  return new_size
+  return tf.stack(tf.unstack(new_size) + [num_channels])
```

Comment:
Isn't unstack followed by a stack really just an append operation? Would it be cleaner to use one op?

Reply:
SGTM